Update ide_debugging.rst (#4669)
qgis/QGIS-Documentation: source/docs/pyqgis_developer_cookbook/plugins/ide_debugging.rst

.. warning:: |outofdate|

A note on configuring your IDE on Linux and Windows
====================================================

> On Linux, all that usually needs to be done is to add the QGIS library locations to the user's `PYTHONPATH` environment variable.  Under most distros, this can be done by editing `~/.bashrc` or `~/.bash-profile` with the following line (tested on OpenSUSE Tumbleweed):

Reviewer: Some rendering suggestions:
* short lines (<=80 characters) are better, simpler to comment and update
* using a single ` character does not render in html so better double it
* for file, let's use :file: directive
* the :: at the end is to make the next line as a code-block
```suggestion
On Linux, all that usually needs to be done is to add the QGIS library locations
to the user's ``PYTHONPATH`` environment variable. Under most distributions,
this can be done by editing :file:`~/.bashrc` or :file:`~/.bash-profile` with
the following line (tested on OpenSUSE Tumbleweed)::
```

> `export PYTHONPATH="$PYTHONPATH:/usr/share/qgis/python/plugins:/usr/share/qgis/python"`

Reviewer: indentation because of code-block
```suggestion
  export PYTHONPATH="$PYTHONPATH:/usr/share/qgis/python/plugins:/usr/share/qgis/python"
```

> Save the file and implement the environment settings by using the following shell command:

Reviewer: ```suggestion
Save the file and implement the environment settings by using the following
shell command::
```

> `source ~/.bashrc`

Reviewer: ```suggestion
  source ~/.bashrc
```

> On Windows, you need to make sure you that you have the same environment

Reviewer: ```suggestion
On Windows, you need to make sure that you have the same environment
```

settings and use the same libraries and interpreter as QGIS. The fastest

> way to do this, is to modify the startup batch file of QGIS.

Reviewer: ```suggestion
way to do this is to modify the startup batch file of QGIS.
```
